rules/windows/registry/registry_set/registry_set_ime_non_default_extension.yml

```diff
@@ -0,0 +1,31 @@
+title: Uncommon Extension In Keyboard Layout Ime File Registry Value
+id: b888e3f2-224d-4435-b00b-9dd66e9ea1f1
+related:
+    - id: 9d8f9bb8-01af-4e15-a3a2-349071530530
+      type: derived
+status: experimental
+description: |
+    Detects usage of Windows Input Method Editor (IME) keyboard layout feature, which allows an attacker to load a DLL into the process after sending the WM_INPUTLANGCHANGEREQUEST message.
+    Before doing this, the client needs to register the DLL in a special registry key that is assumed to implement this keyboard layout. This registry key should store a value named "Ime File" with a DLL path.
+    IMEs are essential for languages that have more characters than can be represented on a standard keyboard, such as Chinese, Japanese, and Korean.
+references:
+    - https://www.linkedin.com/pulse/guntior-story-advanced-bootkit-doesnt-rely-windows-disk-baranov-wue8e/
+author: X__Junior (Nextron Systems)
+date: 2023/11/21
+tags:
+    - attack.defense_evasion
+    - attack.t1562.001
+logsource:
+    product: windows
+    category: registry_set
+detection:
+    selection:
+        TargetObject|contains|all:
+            - '\Control\Keyboard Layouts\'
+            - 'Ime File'
```

Comment:
Shouldn't it be in `Details`?

Reply:
I haven't seen it in the screenshots anywhere. So is it a key or part of the value?

Reply:
![image](https://github.com/SigmaHQ/sigma/assets/8741929/f0f49cc6-c96d-42a1-b9ec-ccfbde8f0ff6)

Reply:
Also its not in the SS of the registry because its supplied via API and is mentioned in the the diagram 
![image](https://github.com/SigmaHQ/sigma/assets/8741929/4e83a73d-46bd-4776-ac04-ac1ce8c45eb1)